bundles/core/org.eclipse.smarthome.core/META-INF/MANIFEST.MF

```diff
@@ -53,7 +53,6 @@ Import-Package: com.google.common.base,
  org.osgi.service.event,
  org.osgi.service.log,
  org.osgi.util.tracker,
- javax.xml.bind,

```

Comment:
The package javax.xml.bind is used by <code>bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/RawType.java</code>
There is this import statement: <code>import javax.xml.bind.DatatypeConverter;</code>